examples/move/scratch_off/sources/math.move

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::math {
+
+    /// Hard coding this as 1 billion
+    const MODULO_FACTOR: u256 = 1_000_000_000;
+
+    public fun bytes_to_u256(hashed_beacon: &vector<u8>): u256 {
+        let output = 0;
+        let bytes_length = 32;
+        let idx = 0;
+        while (idx < bytes_length) {
+            let current_byte = *std::vector::borrow(hashed_beacon, idx);
+            output = (output << 8) | (current_byte as u256) ;
+            idx = idx + 1;
+        };
+        output
+    }
+
+    public fun get_result(random_number: u256): u64 {
+        ((random_number % MODULO_FACTOR) as u64)
+    }
+
+    public fun get_random_u64(hashed_beacon: &vector<u8>): u64 {
+        let output: u256 = 0;
+        let bytes_length: u64 = 32;
+        let idx: u64 = 0;
+        while (idx < bytes_length) {
+            let current_byte = *std::vector::borrow(hashed_beacon, idx);
+            output = (output << 8) | (current_byte as u256) ;
+            idx = idx + 1;
+        };
+        ((output % MODULO_FACTOR) as u64)
```

Comment:
why do you need "% MODULO_FACTOR" here? should_draw_prize does the modulo with the number of tickets and so this one might bias the output

Reply:
We need a modulo factor since the conversion from u256 -> u64 errors if the number is larger than what fits in a u64